src/lua-runtime-fastpm.lua

```diff
@@ -34,12 +34,11 @@ function schema.output_redshifts.action(output_redshifts)
 end
 
 schema.declare{name='omega_m',           type='number', required=true, default=0.3, help='cdm + baryon density parameter at z=0'}
-schema.declare{name='omega_ncdm',           type='number', required=false, default=0.0, help='ncdm density parameter at z=0'}
 schema.declare{name='h',                 type='number', required=true, default=0.7, help="Dimensionless Hubble parameter"}
 schema.declare{name='pm_nc_factor',      type='array:number',  required=true, help="A list of {a, PM resolution}, "}
 schema.declare{name='np_alloc_factor',   type='number', required=true, help="Over allocation factor for load imbalance" }
 schema.declare{name='compute_potential',   type='boolean', required=false, default=false, help="Calculate the gravitional potential."}
-schema.declare{name='m_ncdm',   type='array:number', required=false, default={0.15}, help="Mass of ncdm particles. Enter in descending order."}
+schema.declare{name='m_ncdm',   type='array:number', required=false, default={}, help="Mass of ncdm particles. Enter in descending order."}
```

Comment:
Why does it have to be in descending order? Is it a usual convention?

Reply:
It's not quite correct, really all that is needed is to input the heaviest first. This is because when there are multiple ncdm species, we scale the mass of each species by m_ncdm[0] to give dimensionless quantities r_i=m[i]/m[0]. Having r_i <= 1 samples the FD better for each species, but actually I meant to ask Arka about this, we might actually want r_i >= 1.